README.md

```diff
@@ -110,8 +112,40 @@ The last published version of the package become [available for download](htt
 
 ### Examples
 
-> **Will come soon**
+```js
+var fs = require('fs');
+var promisify = require('promisify-es6')
+
+var IPFS = require('ipfs')
+var node = new IPFS()
 
+function displayVersion() {
+    console.log(node.version());
+}
+
+function initializeRepo() {
+    node.init(function(err){
+        console.log(err);
+    })    
+}
+
+function startDaemon() {
+    node.goOnline(function(msg) {});  
```

Comment:
This should probably call `addFile` in the callback to trigger the interaction once the node is online.